### PR TITLE
Add note field to EntityAttr

### PR DESCRIPTION
**Pull request by syucream**
Add `note` field to EntityAttr to attach human friendly description on each attributes.

image

(...and why it has many formatter diff's...?)

### CODE REVIEW

File: apiclient/typescript-fetch/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmm-com/airone-apiclient-typescript-fetch",
-  "version": "0.0.4",
+  "version": "0.0.5",
```
Comment (syucream) [inline apiclient/typescript-fetch/package.json:3]: @userlocalhost @hinashi could you attach the release label if it looks okay? 🙏

**Comment by hinashi** [comment]
(When do npm run link, the import order of @dmm-com changes...)

**Review by hinashi — APPROVED** [review]
LGTM!

**Review by userlocalhost — APPROVED** [review]
Thank you very much for implementing new feature 👍

**Comment by userlocalhost** [comment]
We need to run following SQL to adopt with this change with previous current schema.
```
--
-- Add field note to entityattr
--
ALTER TABLE `entity_entityattr` ADD COLUMN `note` varchar(200) DEFAULT '' NOT NULL;
ALTER TABLE `entity_entityattr` ALTER COLUMN `note` DROP DEFAULT;
--
-- Add field note to historicalentityattr
--
ALTER TABLE `entity_historicalentityattr` ADD COLUMN `note` varchar(200) DEFAULT '' NOT NULL;
ALTER TABLE `entity_historicalentityattr` ALTER COLUMN `note` DROP DEFAULT;
```

**Pull request merged**